aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. This article covers:
+
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations and compatibility.
+
+## Getting started with .NET native AOT deployment in ASP.NET Core
+
+Native AOT is a publishing option. AOT compilation happens when the app is published. A project opted into Native AOT publishing still uses JIT for F5 debugging, dotnet run, or dotnet build. There are some observable differences:
+
+* Some run time features that aren't compatible with Native AOT are disabled and throw exceptions at run time.
+* A source analyzer is enabled to highlight code that isn't compatible with Native AOT. At publish time, the entire app, including NuGet packages, are analyzed for compatibility again.
```

Comment:
We should show an example of an incompatible API and the experience at build and publish (of what the warnings look like).

Reply:
Will do this in a second pass. I think we need to break this page up. Just like in the main AOT docs there is a sub page for fixing warnings.